src/roman_datamodels/datamodels.py

```diff
@@ -242,10 +242,17 @@ def get_crds_parameters(self):
         -------
         dict
         """
-        return {
+        crds_header = {
             key: val for key, val in self.to_flat_dict(include_arrays=False).items()
             if isinstance(val, (str, int, float, complex, bool))
         }
+        # We need to add two distinct time and date entries derived from the start time
+        # since that is what CRDS expects.
+        time = crds_header['roman.meta.observation.start_time']
```

Comment:
This value won't be included in `crds_header`, since we're only selecting instances of simple built-in types.